cli/go.mod

```diff
@@ -1,4 +1,6 @@
-go 1.20
+go 1.21
```

Comment:
Is this upgrade intended as part of this?

Reply:
I would advocate for taking this on later and doing all happy packages at the same time

Reply:
not sure how that happened, I will revert

Reply:
`gvm` (if you use that) has a tendency to lose track of which go version it's supposed to be on :/